docs/install/installEverest.md

```diff
@@ -33,8 +34,8 @@ We recommend setting up Percona Everest on the [Amazon Elastic Kubernetes Servic
 Before getting started with Percona Everest, we recommend that you:
```

Comment:
Fix the link to Percona Everest CLI

[Percona Everest CLI](https://github.com/percona/percona-everest-cli), which connects Percona Everest to your Kubernetes cluster.